LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-reporting-expected.txt

```diff
@@ -1,12 +1,20 @@
+CONSOLE MESSAGE: Refused to create a TrustedTypePolicy named 'three' because it violates the following Content Security Policy directive: "trusted-types one"
+CONSOLE MESSAGE: [Report Only] Refused to create a TrustedTypePolicy named 'three' because it violates the following Content Security Policy directive: "trusted-types two"
 CONSOLE MESSAGE: Refused to load because it does not appear in the object-src directive of the Content Security Policy.
+CONSOLE MESSAGE: Refused to create a TrustedTypePolicy named 'two' because it violates the following Content Security Policy directive: "trusted-types one"
+CONSOLE MESSAGE: Refused to load because it does not appear in the object-src directive of the Content Security Policy.
+CONSOLE MESSAGE: [Report Only] Refused to create a TrustedTypePolicy named 'one' because it violates the following Content Security Policy directive: "trusted-types two"
+CONSOLE MESSAGE: Refused to load because it does not appear in the object-src directive of the Content Security Policy.
+CONSOLE MESSAGE: SyntaxError: Unexpected token '<'
+
 
 
-Harness Error (TIMEOUT), message = null
+Harness Error (FAIL), message = SyntaxError: Unexpected token '<'
 
-TIMEOUT Trusted Type violation report: creating a forbidden policy. Test timed out
-NOTRUN Trusted Type violation report: creating a report-only-forbidden policy.
-NOTRUN Trusted Type violation report: creating a forbidden-but-not-reported policy.
-NOTRUN Trusted Type violation report: assign string to script url
+PASS Trusted Type violation report: creating a forbidden policy.
+PASS Trusted Type violation report: creating a report-only-forbidden policy.
+PASS Trusted Type violation report: creating a forbidden-but-not-reported policy.
+TIMEOUT Trusted Type violation report: assign string to script url Test timed out
```

Comment:
This seems strange, can you check this?

If this is expected, we usually skip timing out tests until we fix the time out.

Reply:
Just looked and this is expected, this PR only implements one of the two directives this test covers. Should I disable the test until both are done at the cost of less test coverage over this specific PR?

Reply:
Let's do that for now, are you planning to support it soon?
Otherwise, we could update the test to not timeout in the error case.

Reply:
Yeah I plan on supporting that as a follow up PR (slightly larger so might take longer). Will disable this for now